experimenter/experimenter/nimbus_ui_new/filtersets.py

```diff
@@ -0,0 +1,281 @@
+        widget=MultiselectWidget(
+            icon="fa-solid fa-user-shield",
+            attrs={
+                "title": "All QA Statuses",
+                "class": "selectpicker",
+                "data-live-search": "true",
+            },
+        ),
+    )
+    takeaways = django_filters.MultipleChoiceFilter(
+        method="filter_takeaways",
+        choices=NimbusExperiment.Takeaways.choices,
+        widget=MultiselectWidget(
+            icon="fa-solid fa-list-check",
+            attrs={
+                "title": "All Takeaways",
+                "class": "selectpicker",
+                "data-live-search": "true",
+            },
+        ),
+    )
+    owner = django_filters.ModelMultipleChoiceFilter(
+        queryset=User.objects.order_by("email"),
+        widget=MultiselectWidget(
+            icon="fa-solid fa-users",
+            attrs={
+                "title": "All Owners",
+                "class": "selectpicker",
+                "data-live-search": "true",
+            },
+        ),
+    )
+    subscribers = django_filters.ModelMultipleChoiceFilter(
+        queryset=User.objects.order_by("email"),
+        widget=MultiselectWidget(
+            icon="fa-solid fa-bell",
+            attrs={
+                "title": "All Subscribers",
+                "class": "selectpicker",
+                "data-live-search": "true",
+            },
+        ),
+    )
+
+    class Meta:
+        model = NimbusExperiment
+        fields = [
+            "status",
+            "search",
+            "type",
+            "application",
+            "channel",
+            "firefox_min_version",
+            "feature_configs",
+            "countries",
+            "languages",
+            "locales",
+            "targeting_config_slug",
+            "projects",
+            "qa_status",
+            "takeaways",
+            "owner",
+            "subscribers",
+        ]
+
+    def filter_status(self, queryset, name, value):
+        match value:
+            case StatusChoices.REVIEW:
+                return queryset.filter(
+                    status=NimbusExperiment.Status.DRAFT,
+                    publish_status=NimbusExperiment.PublishStatus.REVIEW,
+                )
+            case StatusChoices.ARCHIVED:
+                return queryset.filter(is_archived=True)
+            case StatusChoices.MY_EXPERIMENTS:
+                return queryset.filter(owner=self.request.user)
+            case _:
+                return queryset.filter(
+                    status=value,
+                    is_archived=False,
+                ).exclude(publish_status=NimbusExperiment.PublishStatus.REVIEW)
+
+    def filter_search(self, queryset, name, value):
+        search_fields = Concat(
+            "name",
+            "slug",
+            "public_description",
+            "hypothesis",
+            "takeaways_summary",
+            "qa_comment",
+            output_field=models.TextField(),
```

Comment:
do we want to add features in a search too?

Reply:
I thought about that but then you'd have to do a join and slow this down a lot, and there's already the feature config drop down filter.